test/tools/fuzzer_common.cpp

```diff
@@ -78,7 +78,7 @@ void FuzzerUtil::testCompilerJsonInterface(string const& _input, bool _optimize,
 void FuzzerUtil::forceSMT(StringMap& _input)
 {
 	// Add SMT checker pragma if not already present in source
-	static const char* smtPragma = "pragma experimental SMTChecker;";
+	auto static constexpr smtPragma = "pragma experimental SMTChecker;";
```

Comment:
I think the codebase tends to do `static auto` instead of `auto static`, although I can't find an explicit rule from https://github.com/ethereum/solidity/blob/develop/CODING_STYLE.md

Similarly for evmhost.cpp